lib/gateway/service-map.js

```diff
@@ -201,7 +201,7 @@ async function buildServiceMap (services, errorHandler) {
       serviceConfig = await serviceMap[service.name].init()
     } catch (err) {
       serviceConfigErr = err
-      errorHandler(err, service)
+      if (!service.mandatory) errorHandler(err, service)
```

Comment:
we should probably log if it's not mandatory

Reply:
Ok I will add a logline before calling the error handler for non mandatory services